ConsistencyManagerTests/PauseListenerTests.swift

```diff
@@ -18,9 +18,9 @@ class PauseListenerTests: ConsistencyManagerTestCase {
     /**
      This provides some reusable code for generating a test model with many children.
      */
-    func setUpListeners(quantity: Int) -> ([TestListener], ConsistencyManager) {
+    func setUpListeners(quantity: Int, testProjections: Bool) -> ([TestListener], ConsistencyManager) {
```

Comment:
this is really small...but testProjections kind of sounds like a name for a set of test projections.
Maybe shouldTestProjections would be a more clear name for when it's passed into methods.